src/main/java/frc/robot/constants/RobotComponents.java

```diff
@@ -112,6 +112,16 @@
                 withClosedLoopRampRate(0.5).
                 withPID(RobotConstants.LOCAL_CONSTANTS.localIntakeOpenerConstants.pidfCoefs);
         public static PIDFTalonSRX MOTOR = new PIDFTalonSRX(
-                CAN.MOTOR_ID.INTAKE_OPENER_MOTOR_ID, INTAKE_OPENER_MOTOR_CONFIG);
+                CAN.IntakeOpener.INTAKE_OPENER_MOTOR_ID, INTAKE_OPENER_MOTOR_CONFIG);
+    }
+
+    protected static class PitcherComponents {
+        private static final MotorConfig PITCHER_MOTOR_CONFIG = new MotorConfig().
```

Comment:
Remove PITCHER_ in line 119.

Reply:
but in intake opener is writine like that so sould we chance intake also ?

Reply:
Yes

Reply:
ok